src/ddmd/staticcond.d

```diff
@@ -37,6 +37,39 @@ import ddmd.utils;
 
 bool evalStaticCondition(Scope* sc, Expression exp, Expression e, ref bool errors)
 {
+    if (e.op == TOKandand)
+    {
+        AndAndExp aae = cast(AndAndExp)e;
+        bool result = evalStaticCondition(sc, exp, aae.e1, errors);
+        if (errors || !result)
+            return false;
+        result = evalStaticCondition(sc, exp, aae.e2, errors);
+        return !errors && result;
+    }
+
+    if (e.op == TOKoror)
+    {
+        OrOrExp ooe = cast(OrOrExp)e;
+        bool result = evalStaticCondition(sc, exp, ooe.e1, errors);
+        if (errors)
+            return false;
+        if (result)
+            return true;
+        result = evalStaticCondition(sc, exp, ooe.e2, errors);
+        return !errors && result;
+    }
+
+    if (e.op == TOKquestion)
+    {
+        CondExp ce = cast(CondExp)e;
+        bool result = evalStaticCondition(sc, exp, ce.econd, errors);
+        if (errors)
+            return false;
+        Expression leg = result ? ce.e1 : ce.e2;
+        result = evalStaticCondition(sc, exp, leg, errors);
+        return !errors && result;
+    }
+
```

Comment:
There are no paren expressions in the AST. `(a+b)` is the same AST as `a+b`. Parens are for grouping.